resolver/resolver.cc

```diff
@@ -1289,7 +1289,7 @@ class ResolveConstantsWalk {
         if (checkAmbiguousDefinition(ctx)) {
             const auto ambigDef = findAnyDefinitionAmbiguousWithCurrent(ctx);
             if (ambigDef.exists()) {
-                if (auto e = ctx.beginError(original.loc, core::errors::Resolver::AmbiguousDefinitionError)) {
+                if (auto e = ctx.beginError(original.declLoc, core::errors::Resolver::AmbiguousDefinitionError)) {
                     e.setHeader("Class definition is ambiguous");
```

Comment:
Could we show the ambiguous name also in the header? Right now it only shows up in the error lines.

Reply:
I had meant to break this commit out into a separate PR, do you mind if we push this to a follow-up task?

Reply:
wfm